spec/index.md

```diff
@@ -2855,7 +2915,7 @@ A reference implementation would likely maintain a separate list of `messages` f
 
 #### API requests
 
-We distinguish between the *asynchronous* API requests (type `Request`) passed to `/api/v2/…/call`, which may be present in the IC state, and the *synchronous* API requests passed to `/api/v2/…/read_state` and `/api/v2/…/query`, which are only ephemeral.
+We distinguish between the *asynchronous* API requests (type `Request`) passed to `/api/v2/…/call` and `/api/v2/…/sync_call`, which may be present in the IC state, and the *synchronous* API requests passed to `/api/v2/…/read_state` and `/api/v2/…/query`, which are only ephemeral.
```

Comment:
This is slightly confusing: above (and in the API name) we explicitly call `sync_call` "synchronous". Yet here we categorize it as asynchronous.

Is there any possibility to use a different categorization? If not, maybe the endpoint could be named `call_and_await`.

Reply:
> Yet here we categorize it as asynchronous.

It is asynchronous in the worst-case.

Reply:
But I agree that it'd make sense to make this more precise and ideally also formally define the response to sync_call on the happy path: a certificate similar to the case of read state request responses.

Reply:
[Changed ](https://github.com/dfinity/interface-spec/pull/265/commits/07a46146730e44a1f8ffd5a472c63f3116c81d44)

Reply:
SGTM

I'm just wondering whether "call" or "update call" is the most precise term. "Call" seems too generic, since queries are also calls. And "update call" is too specific, as (I believe that) replicated queries are not technically update calls.

Maybe we should use "replicated call", which should cover all ingress messages. Although I suppose that certified queries could also be claimed to be "replicated calls".

That being said, the above is mostly nitpicking on my part. So feel free to ignore.

Reply:
> replicated queries are not technically update calls.

I believe they are so changed to "update calls".